import "dna.proto";
import "answer.proto";

package ProblemBuffers;

message Problem {
  optional ProblemSettings settings = 1;
  repeated AnswerBuffers.NetConfig configs = 2;
  optional RemyBuffers.WhiskerTree whiskers = 3;
}

message ProblemSettings {
  optional uint32 prng_seed = 11;
  optional bool trace = 12;
  optional uint32 carefulness = 13;
}
